package/batocera/ports/yquake2/embedded.cfg

```diff
@@ -0,0 +1,4 @@
+set s_openal "0"
```

Comment:
what does this cfg file do? and where does it apply?

Reply:
ok so again this should be done in configgen to apply for certain hardware.
i'm sure higher-end arm boards don't need this also.